src/sentry/conf/server.py

```diff
@@ -2527,3 +2527,5 @@ def build_cdc_postgres_init_db_volume(settings):
 # Note: The docs have this backwards. We set this to False here so that we always add check
 # constraints instead of setting the column to not null.
 ZERO_DOWNTIME_MIGRATIONS_USE_NOT_NULL = False
+
+ANOMALY_DETECTION_URL = "127.0.0.1:9091"
```

Comment:
Outside the scope of this PR, but should make sure you set the corresponding setting for production when you're ready to ship